_includes/package_links.html

```diff
@@ -41,16 +41,6 @@
       {% assign is_ros1_page = true %}
     {% endif %}
   {% endfor %}
-  
-  {% if is_ros1_page %}
```

Comment:
The code below just adds a link on ROS1 pages to the wiki for a package, which is still useful documentation for ROS1 packages. If you look at package download counts (see for June 2025 https://rkent.github.io/ros_webdata/package_downloads-2025-06.json), ROS1 downloads, mostly noetic, are still about 1/4 of ROS2 downloads. So ROS1 packages are still very active in the community.

I can see the value of degrading ROS1 support by, for example with this PR, stopping scraping the wiki. But as long as rosindex continues to support ROS1 packages, and if the wiki remains alive, then there is no reason to remove the link here. Is the plan to take the wiki offline?

Reply:
It's a bit complicated, The wiki is now officially EOL, and we're going to be transitioning it to a static archive at some point to try to support deep links. And we'll also want to do a process to propose/migrate all the package specific content to README.md files in the repos. But since this will only be on the older repos and likely many of them are unmaintained it will have minimal impact to keep it here so I've added it back.